todo-app/package.json

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "todo-app",
+  "name": "week1",
```

Comment:
Usually the name of our project is going to stay the same. If you're interested in incrementing based on week1, week2, etc, look into incrementing the `version` value using [SEMVER](https://semver.org/).